kubernetes/types.go

```diff
@@ -96,7 +96,7 @@ const (
 	serviceMeshRbacConfigType     = "ServiceMeshRbacConfig"
 	serviceMeshRbacConfigTypeList = "ServiceMeshRbacConfigList"
 
-	// Authorization Policies
+	// Authorization PeerAuthentications
```

Comment:
Perhaps this comment is misleading but no blocker just a note to change it in the future.